source/trackers.json

```diff
@@ -1098,6 +1098,7 @@
         "amazonbrowserapp.es": "amazon",
         "amazoncrl.com": "amazon",
         "firetvcaptiveportal.com": "amazon",
+        "pv-cdn.net": "amazon",
```

Comment:
could be in Amazon Video as well 
https://github.com/AdguardTeam/companiesdb/pull/299#discussion_r1378993742